### PR TITLE
EZP-28229: As an Editor, I want to have styled login form

### PR DESCRIPTION
**Pull request by dew326**
**Jira ticket:** https://jira.ez.no/browse/EZP-28229

**Screenshot:**
login-form

### CODE REVIEW

File: src/bundle/Resources/views/Security/login.html.twig [hunk 1]
```diff
@@ -2,33 +2,37 @@
...
-                            <button type="submit" class="btn btn-primary">{{ 'authentication.submit'|trans|desc('Login') }}</button>
-                        </fieldset>
-                    </form>
-                </div>
+                        <button type="submit" class="btn btn-primary ez-btn ez-btn--login">{{ 'authentication.submit'|trans|desc('Login') }}</button>
+                    </fieldset>
+                </form>
+            </div>
+            <div class="ez-login__footer">
+                <span>eZ Platform ©1999-2017 <a href="http://ez.no">eZ Systems AS</a> and others.</span>
```
Comment (adamwojs) [inline src/bundle/Resources/views/Security/login.html.twig]: Can we use `{{ "now"|date("Y") }}` instead of hardcoding current year ? 😉

Comment (lserwatka) [reply src/bundle/Resources/views/Security/login.html.twig]: Yes, this is good idea!

Comment (dew326) [reply src/bundle/Resources/views/Security/login.html.twig]: I didn't know how to do it in PHP. I will add it ;)

**Comment by dew326** [comment]
@adamwojs @sunpietro done

File: src/bundle/Resources/public/scss/_login.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        color: $ez-white;
+
+        &-wrapper {
+            width: 18rem;
+            height: 90%;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+
+            .ez-login__form-label {
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_login.scss]: Does it have to be nested like this? I think can be placed one level up without any issues :)

File: src/bundle/Resources/public/scss/_login.scss [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+                margin-bottom: 0;
+                margin-top: 1rem;
+            }
+
+            .ez-login__form-input {
```
Comment (sunpietro) [inline src/bundle/Resources/public/scss/_login.scss]: Same here.

**Comment by m-tyrala** [comment]
QA test ok.

**Pull request merged**